examples/with_airflow/with_airflow/assets_repository.py

```diff
@@ -40,6 +39,8 @@ def new_upstream_asset_2():
 )
 
 
-@repository
+@repository(
+    top_level_resources={"airflow_db": make_ephemeral_airflow_db_resource()},
```

Comment:
oh lord don't use this. @benpankow  we should make this `_top_level_resources` so indicate internal-only?

@Ramshackle-Jamathon move this to `Definitions` please

Reply:
with_airflow is such a mess atm